travis.cfg

```diff
@@ -1,7 +1,8 @@
 [buildout]
 extends =
     https://raw.github.com/collective/buildout.plonetest/master/buildout-cache.cfg
-    test-plone-4.x.cfg
```

Comment:
this line should be left as test-plone-4.x.cfg, so that the sed command in .travis.yml works.  I guess you ran travis commands locally & it changed it for you!